### ISSUE TITLE
[win32-openssh] "AddPrivileges" PowerShell exception during installation on Windows 2008

### ISSUE THREAD

**Issue by C-Duv** [DarwinJS/ChocoPackages#3]
Version 2016.05.15 fails to install (on Windows 2008).

Ran the following command:
```
choco install win32-openssh --version 2016.05.15 -params '"/SSHServerFeature"'
```

And got:
```
[...]
 Everything is Ok
 Files: 15
 Size:       11095661
 Compressed: 5174944
 C:\Users\ADMINI~1\AppData\Local\Temp\2\OpenSSHTemp
 PATH environment variable does not have C:\Program Files\OpenSSH-Win64 in it. A
dding...
 AVERTISSEMENT : You have specified SSHServerFeature - this machine is being
 configured as an SSH Server including opening port 22.
 AVERTISSEMENT : Explicitly disabling sshd logging as it currently logs about .5

  GB / hour
 Generating sshd keys in "C:\Program Files\OpenSSH-Win64"
 C:\Program Files\OpenSSH-Win64\ssh-keygen.exe: generating new host keys: RSA DS
A ECDSA ED25519
 Ok.
 [SC] SetServiceObjectSecurity réussite(s)
 AVERTISSEMENT : The following private keys should be removed from the machine:

 ssh_host_dsa_key ssh_host_rsa_key ssh_host_ecdsa_key ssh_host_ed25519_key
 [SC] ChangeServiceConfig réussite(s)
 String account: 30
 Exception calling "AddPrivileges" with "2" argument(s): "Paramètre incorrect"
 Au niveau de C:\ProgramData\chocolatey\lib\win32-openssh\tools\chocolateyinstal

 l.ps1 : 214 Caractère : 49
 +   [MyLsaWrapper.LsaWrapperCaller]::AddPrivileges <<<< ("NT SERVICE\SSHD", "Se

 ServiceLogonRight")
     + CategoryInfo          : NotSpecified: (:) [], ParentContainsErrorRecordE

    xception
     + FullyQualifiedErrorId : DotNetMethodException
The install of win32-openssh was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\win32-openssh\tools\chocolate
yinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 package(s). 1 package(s) failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Failures:
 - win32-openssh
```

Service `ssh-agent` and `sshd` seems installed, `ssh-agent` is running but service `ssh` fails to start: 
> Error 1068: The dependency service or group failed to start.

**Comment by dave-miles** [comment]
Also Windows 7 Enterprise.
A work around is to finish the install by hand:

```bat
cd c:\Program Files\OpenSSH-Win64
ntrights.exe -u "NT SERVICE\SSHD" +r SeAssignPrimaryTokenPrivilege
net start sshd
```

Note that chocolatey will mark the install as failed so **choco list -lo** will not show the package.

**Comment by DarwinJS** [comment]
I originally had it using ntrights.exe - but it is not compatible with Nano - so looking for a native way to handle the rights assignment.

**Comment by C-Duv** [comment]
I can confirm @dave-miles 's commands works.

**Comment by DarwinJS** [comment]
Version 2016.04.15.201601116 is in moderation and uses a new approach for add-privileges if you wish to do early testing.

**Comment by DarwinJS** [comment]
Fixed in 2016.05.30 - uses ntrights.exe when powershell is less than 3.0.  Tested on Server 2008 R2 RTM.
